### ISSUE TITLE
structured content

### ISSUE THREAD

**Issue by flosse** [cobalt-org/cobalt.rs#91]
let's say I'd like to render a page with several areas like this:

``` liquid
<section>
  <div id="sidebar">
    {% for section in sidebar.sections %}
      <section>{{ section.content }}
        <footer>{{ section.footer }}</footer>
      </section>
    {% endfor %}
  </div>
  <div id="content">
    <article>
      <header>{{ header }}</header>
      {{ content}}
    </article>
  </div>
</section>
```

How can I define my content in eg. `mypage.md`?

**Comment by epage** [comment]
I wonder if one way of resolving this is to implement support for [data files](https://jekyllrb.com/docs/datafiles/).

**Comment by epage** [comment]
Also, I suspect this and #89 could be solved with the same tool.

**Comment by epage** [comment]
Does #292 solve this for you?

**Comment by epage** [comment]
Closing as fixed by #256 unless the filer would like to contest that.

**Comment by epage** [comment]
Oops, forgot to close this.

Options now:
- data files available as `site.data.<path>`
- `site: data` field in `_cobalt.yml` available as `site.data`
- Default frontmatter in `_cobalt.yml` available as `page.data`.
